src/content/reference/okteto-manifest.mdx

```diff
@@ -274,6 +279,59 @@ deploy:
 - `virtualServices`: a list of virtual services to divert. Each virtual service is defined with it's name, namespace, and an optional list of routes to be diverted. By default, all routes are diverted.
 - `hosts`: the list of hosts you want to divert in the developer namespace. Requests to these virtual services will have the header `baggage.okteto-divert` injected.
 
+### test (object, optional)
+
+A dictionary of tests to execute tests using okteto infrastructure via [Remote Execution](core/remote-execution.mdx#how-remote-execution-works)
+
+```yaml
+test:
+  setup:
+    image: okteto/golang:1
+    commands:
+      - "echo prepare tests"
+  unit:
+    image: okteto/golang:1
+    context: tests
+    depends_on:
+      - setup
+    artifacts:
+      - test-results.xml
+      - coverage.out
+    caches:
+      - /go
+      - /root/.cache
+    commands:
+      - "go test . -v"
+```
+
+- `[name]`: the name of the test section (user defined)
+
+#### image (string, optional)
+
+The base image used to run your test.
+
+If it is not defined, okteto uses the [pipeline-runner](https://github.com/okteto/pipeline-runner) image unless an Okteto Cluster admin specifically changed it [here](self-hosted/helm-configuration.mdx/#installer) with a [custom installer image](admin/custom-installer-image.mdx)
```

Comment:
I think build is broken because of this link:
```
8:14:43 PM:   Exhaustive list of all broken links found:
8:14:43 PM:   - Broken link on source page path = /docs/1.21/reference/okteto-manifest/:
8:14:43 PM:      -> linking to self-hosted/helm-configuration.mdx/#installer (resolved as: /docs/1.21/reference/okteto-manifest/self-hosted/helm-configuration.mdx/#installer)
8:14:43 PM:       at throwError (/opt/build/repo/node_modules/@docusaurus/logger/lib/index.js:79:11)
8:14:43 PM:       at reportBrokenLinks (/opt/build/repo/node_modules/@docusaurus/core/lib/server/brokenLinks.js:242:47)
8:14:43 PM:       at handleBrokenLinks (/opt/build/repo/node_modules/@docusaurus/core/lib/server/brokenLinks.js:274:5)
8:14:43 PM:       at buildLocale (/opt/build/repo/node_modules/@docusaurus/core/lib/commands/build.js:192:47)
8:14:43 PM:       at async tryToBuildLocale (/opt/build/repo/node_modules/@docusaurus/core/lib/commands/build.js:46:20)
8:14:43 PM:       at async mapAsyncSequential (/opt/build/repo/node_modules/@docusaurus/utils/lib/jsUtils.js:44:24)
8:14:43 PM:       at async Command.build (/opt/build/repo/node_modules/@docusaurus/core/lib/commands/build.js:82:21)
```
You probably need to remove the `/` between `helm-configuration.mdx` and `#installer`